jenkins-instance.yaml

```diff
@@ -0,0 +1,7 @@
+--
```

Comment:
We already have `deploy/crds/jenkins_v1alpha1_jenkins_cr.yaml` for `example-jenkins` can we continue using that itself  ?